addon/components/sl-select.js

```diff
@@ -103,7 +103,24 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
     destroySelect2: Ember.on(
         'willClearRender',
         function() {
-            this.input.off( 'change' ).select2( 'destroy' );
+            this.get( 'input' ).off( 'change' ).select2( 'destroy' );
```

Comment:
Doesn't need namespace referencing?  Know you are doing that in a separate PR but are the merges going to be clean between it and this one?